templates/solutions/ai-infrastructure/index.html

```diff
@@ -252,16 +252,16 @@ <h2>Accelerate innovation with certified hardware</h2>
           Get peace of mind with Ubuntu certified hardware from all major OEMs. Choosing validated platforms significantly reduces deployment time and costs for end-customers, in addition to improving reliability and providing bug-fixes faster.
         </p>
         <p class="p-section--shallow">
-          Combined with NVIDIA DGX systems or NVIDIA-Certified systems, choosing an Ubuntu-certified platform provides secure, performant hardware that is guaranteed to deploy quickly and run smoothly.
+          For example, combined with NVIDIA DGX systems or NVIDIA-Certified systems, choosing an Ubuntu-certified platform provides secure, performant hardware that is guaranteed to deploy quickly and run smoothly.
         </p>
         <hr class="is-muted" />
         <p>
-          <a href="/partners/silicon">Explore our silicon programme&nbsp;&rsaquo;</a>
+          <a href="https://ubuntu.com/certified">Explore certified hardware&nbsp;&rsaquo;</a>
```

Comment:
For some reason, on the demo 'Explore our certified hardware'.  Or am I being really dumb hahaha
![image](https://github.com/canonical/canonical.com/assets/58276363/f022cdbf-6284-421a-a838-18c6e4542d13)